.all-contributorsrc

```diff
@@ -4,18 +4,25 @@
   "repoType": "github",
   "repoHost": "https://github.com",
   "files": [
-    "README.md"
+    "README.md",
+    "ui/docs/people.md"
   ],
   "imageSize": 100,
   "commit": false,
+  "types": {
+    "staff": {
+      "symbol": "👷",
+      "description": "Current or former Taskcluster staff member"
```

Comment:
I'm sorta less interested in having former staff members on here. Part of the point to me is a list of people who can be asked questions, etc. Maybe we need an alumni type as well? This is a tiny nit though, either way is fine.

Reply:
Good point.. what's the emoji for that?